### PR TITLE
changes in navigation bar

### PR DESCRIPTION
**Pull request by rajveer1309**
changed the tittle of website ,removed two columns events and chiefs f…rom navigation bar and removed their body too. removed the language button from the top and edited the word restrauntly to bubby bakery from the beginning of the page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8" />
     <meta content="width=device-width, initial-scale=1.0" name="viewport" />
 
-    <title>Restaurantly Bootstrap Template - Index</title>
+    <title>A home of delicous cakes</title>
```
Comment (bllackjack) [inline index.html]: 1. We need to fix the spelling of delicious.
2. We need to add Bubby Bakery as a prefix to this line. 'Bubby Bakery - A home of delicious cakes'

File: index.html [hunk 2]
```diff
@@ -60,19 +60,12 @@
       >
         <div class="contact-info d-flex align-items-center">
           <i class="bi bi-phone d-flex align-items-center"
-            ><span>+91 9876543210</span></i
+            ><span>+91 9876782011</span></i
```
Comment (bllackjack) [inline index.html]: We should probably also add hyphens (-) somewhere in the number. Take reference from other websites on how to print an Indian phone number.

File: index.html [hunk 3]
```diff
@@ -60,19 +60,12 @@
...
           >
           <i class="bi bi-clock d-flex align-items-center ms-4"
-            ><span> Mon-Sat: 11AM - 23PM</span></i
+            ><span> Mon-Sat: 7AM - 9:30PM</span></i
```
Comment (bllackjack) [inline index.html]: Is Bubby Bakery closed on Sunday?

File: index.html [hunk 4]
```diff
@@ -98,31 +91,18 @@ <h1 class="logo me-auto me-lg-0">
             <li><a class="nav-link scrollto" href="#about">About</a></li>
             <li><a class="nav-link scrollto" href="#menu">Menu</a></li>
             <li><a class="nav-link scrollto" href="#specials">Specials</a></li>
-            <li><a class="nav-link scrollto" href="#events">Events</a></li>
-            <li><a class="nav-link scrollto" href="#chefs">Chefs</a></li>
             <li><a class="nav-link scrollto" href="#gallery">Gallery</a></li>
             <li class="dropdown">
               <a href="#"
-                ><span>Drop Down</span> <i class="bi bi-chevron-down"></i
+                ><span>Other items</span> <i class="bi bi-chevron-down"></i
```
Comment (bllackjack) [inline index.html]: We need to think of another name, Other items doesn't look nice. Ask the client.

File: index.html [hunk 5]
```diff
@@ -133,7 +113,7 @@ <h1 class="logo me-auto me-lg-0">
         <a
           href="#book-a-table"
           class="book-a-table-btn scrollto d-none d-lg-flex"
-          >Book a table</a
+          >Book a Cake</a
```
Comment (bllackjack) [inline index.html]: Do we book a cake? I'm not really sure. I think it should be 'Order a Cake'

File: index.html [hunk 6]
```diff
@@ -148,8 +128,8 @@ <h1 class="logo me-auto me-lg-0">
       >
         <div class="row">
           <div class="col-lg-8">
-            <h1>Welcome to <span>Restaurantly</span></h1>
-            <h2>Delivering great food for more than 18 years!</h2>
+            <h1>Welcome to <span>Bubby Bakers</span></h1>
```
Comment (bllackjack) [inline index.html:123]: Are we sure we wanna go with Bubby bakers instead of Bubby Bakery?

**Comment by bllackjack** [comment]
I don't see the images being deleted which are not used? We don't want to keep unnecessary images that increase the size of this website. Please delete them as well and update this pull request by making another commit in this branch.

File: index.html (hunk 1 above)
Comment (rajveer1309) [reply index.html]: ok ill go though it

File: index.html (hunk 2 above)
Comment (rajveer1309) [reply index.html]: hanji i added hyphen instead of space

File: index.html (hunk 4 above)
Comment (rajveer1309) [reply index.html]: i changed to sunday

**Pull request merged**